cybsi/api/observation/api.py

```diff
@@ -137,6 +138,22 @@ def search(
         page = Page(self._connector.do_get, resp, ObservationHeaderView)
         return page
 
+    def view(self, observation_uuid: uuid.UUID) -> "ObservationHeaderView":
+        """Get observation header.
+
+        Note:
+            Calls `GET /enrichment/observations/{observation_uuid}`.
+        Args:
+            observation_uuid: Observation uuid.
+        Returns:
+            Header of the observation.
+        Raises:
+            :class:`~cybsi.api.error.NotFoundError`: Observation not found.
```

Comment:
Минор. А давай здесь добавим ?
```suggestion
            :class:`~cybsi.api.error.NotFoundError`: Observation header not found.
```

Reply:
Не вижу в этом смысла. Если мы говорим про заголовок наблюдения, то должно быть и само наблюдение. Поэтому решил не специфицировать, что заголовок не найден - не найдено всё наблюдение (чей заголовок запрашивают)

Reply:
Убедил) Давай оставим так